open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/FeatureActionsController.java

```diff
@@ -61,7 +55,7 @@
     private final ExecutorService myExecutor = ThreadUtilities.newTerminatingFixedThreadPool("FeatureActionsController", 1);
 
     /** Executor used to handle style registry changes. */
-    private final transient Executor myProcrastinatingExecutor = CommonTimer.createProcrastinatingExecutor(200);
+//    private final transient Executor myProcrastinatingExecutor = CommonTimer.createProcrastinatingExecutor(200);

```

Comment:
did you mean to leave all these comments ?

Reply:
I believe we need to keep this and I don't think this is the cause of the ticket.